src/crimson/osd/pg_backend.cc

```diff
@@ -1234,6 +1234,26 @@ PGBackend::interruptible_future<> PGBackend::omap_remove_range(
   return seastar::now();
 }
 
+PGBackend::interruptible_future<> PGBackend::omap_remove_key(
+  ObjectState& os,
+  const OSDOp& osd_op,
+  ceph::os::Transaction& txn)
+{
+  ceph::bufferlist to_rm_bl;
+  try {
+    auto p = osd_op.indata.cbegin();
+    decode_str_set_to_bl(p, &to_rm_bl);
+  } catch (buffer::error& e) {
+    throw crimson::osd::invalid_argument{};
```

Comment:
Another established practice. Maybe we should follow with a clean-up that errorates the stuff.